Input/IME/WinForms/WinFormsIMEHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ImeSharp;
+
+using Microsoft.Xna.Framework;
+
+namespace Rampastring.XNAUI.Input.IME.WinForms
+{
+    /// <summary>
+    /// Integrate IME to XNA framework.
+    /// </summary>
+    internal class WinFormsIMEHandler : IMEHandler
+    {
+        public WinFormsIMEHandler(Game game)
+        {
+            InputMethod.Initialize(game.Window.Handle);
+            InputMethod.TextInputCallback = OnTextInput;
+            InputMethod.TextCompositionCallback = (compositionText, cursorPosition, _, _, _, _) =>
+            {
+                Composition = compositionText.ToString();
+                CompositionCursorPos = cursorPosition;
+            };
+        }
+
+        public override bool Enabled
+        {
+            get => InputMethod.Enabled;
+            protected set => InputMethod.Enabled = value;
+        }
```

Comment:
IIRC properties should be before the constructor, @Rampastring correct me if I am wrong.